index.js

```diff
@@ -1,12 +1,156 @@
-const express = require('express');
-const helmet = require('helmet');
-
+const express = require("express");
+const helmet = require("helmet");
+const knex = require("knex");
+const knexConfig = require("./knexfile.js");
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
+// Connect to the Configured DataBase
+
+const db = knex(knexConfig.development);
+
 // endpoints here
+server.get("/", (req, res) => {
+  res.send("api working");
+});
+
+// get all zoos
+server.get("/api/zoos", (req, res) => {
+  db("zoos")
+    .then(zoos => {
+      res.status(200).json(zoos);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+// post to zoos
+server.post("/api/zoos", (req, res) => {
```

Comment:
Another option for doing your routes is using the [`route`](https://expressjs.com/en/4x/api.html#app.route) method:
```javascript
server.route('/api/zoos')
  .get((req, res) => {
    db('zoos')
      .then(zoos => {
        res.status(200).json(zoos);
      })
      .catch(err => {
        res.status(500).json(err);
      });
  })
  .post((req, res) => {
    if (!req.body.name) {
      res.status(400).json({ error: 'Provide Zoo Name' });
    } else {
      db('zoos')
        .insert(req.body)
        .then(id => {
          res.status(201).json(id);
        })
        .catch(err => {
          res.status(500).json(err);
        });
    }
  });
```